02. 1, 2차원 배열 탐색/03. 가위바위보/백지원/index.js

```diff
@@ -1,7 +1,21 @@
 function solution(a, b) {
-  let answer = '';
+  let answer = [];
 
-  return answer;
+  function whoWin(a, b) {
+    if (a === b) return "D";
+    if ((a === 1 && b === 3) || (a === 2 && b === 1) || (a === 3 && b === 2)) {
+      return "A";
+    } else {
+      return "B";
+    }
```

Comment:
앞에서 얼리리턴을 하고 있으니 else문 없이 return 해도 될 거 같아요~
```suggestion
    return "B";
```

Reply:
그렇네요 수정했습니다 감사합니당!